src/components/timeline/timeline-event/timeline-event.component.html

```diff
@@ -1,5 +1,5 @@
 <li class="timeline-inverted">
-    <div class="timeline-badge {{ badgeColor }}">
+    <div [ngClass]="['timeline-badge', badgeColor]">

```

Comment:
We can use the `class` attribute for any permanent classes and use ngClass for the dynamic class, eg:

```html
<div class="timeline-badge" [ngClass]="badgeColor">
```